src/app/components/TransactionsTable/TransactionModal.tsx

```diff
@@ -75,8 +75,8 @@ export default function TransactionModal({
               className={classNames(
                 "text-3xl font-medium",
                 transaction.type == "received"
-                  ? "text-green-600 dark:color-green-400"
-                  : "text-orange-600 dark:color-orange-400"
+                  ? "text-green-600 dark:text-emerald-500"
```

Comment:
@reneaaron did you have any thoughts on the outgoing and incoming colors? It would be nice not to hard code them (I guess these are leaked in multiple components now)

Reply:
We could define colors for these ones of course, I think there are more important things to tackle right now though.